index.html

```diff
@@ -18,18 +22,22 @@
     <link rel="stylesheet" href="firstSectionStyle.css" />
     <link rel="stylesheet" href="secondSectionStyle.css" />
     <link rel="stylesheet" href="carouselImages.css" />
-    <link rel="stylesheet" href="commentsSection.css">
+    <link rel="stylesheet" href="testimonialsSection.css" />
     <link rel="stylesheet" href="footer.css" />
+    <script
+      src="https://code.jquery.com/jquery-3.7.1.min.js"
+      integrity="sha256-/JqT3SQfawRcv/BIHPThkBvs0OEvtFFmqPF/lYI/Cxo="
+      crossorigin="anonymous"
+    ></script>
   </head>
   <body>
-    <nav class="navbar navbar-expand-lg nav-bar-light bg-light px-5">
+    <nav class="navbar navbar-expand-lg nav-bar-light bg-light px-3 px-md-5">
       <div class="container-fluid">
         <a href="#" class="navbar-brand"
           ><img
             class="logo-nav"
             src="./img/invariaLogo.svg"
```

Comment:
Always tend to use absolute path over relative path in html. You can find an example and more info [here](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Web_mechanics/What_is_a_URL#examples_of_absolute_urls) (Implicit domain name)

```suggestion
            src="img/invariaLogo.svg"
```

Reply:
Corrected